omero_marshal/legacy/affinetransform.py

```diff
@@ -78,6 +89,9 @@ def __str__(self):
             unwrap(self._a12),
             ]))
 
+    def get_svg_transform(self):
```

Comment:
If we're going to store this and make it available in this manner maybe use a property, document it as public API and always make it available?